src/components/statsBar/StatsView.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
```

Comment:
This is the only file we need to publish. The rest can stay in respective projects